include/sunlinsol/sunlinsol_ginkgobatch.hpp

```diff
@@ -268,9 +268,9 @@ class BatchLinearSolver : public sundials::impl::BaseLinearSolver,
       col_scale_vec_ =
         std::move(impl::WrapBatchScalingArray(GkoExec(), num_batches_, s1_));
 
-      if (!s2inv_.Convert())
+      if (!s2inv_.get())
       {
-        s2inv_ = sundials::experimental::NVectorView(N_VClone(s2));
+        s2inv_ = std::move(sundials::experimental::N_VectorView(N_VClone(s2)));
```

Comment:
I think the `std::move` is unnecessary since the things being moved are already r-values.

```suggestion
        impl::WrapBatchScalingArray(GkoExec(), num_batches_, s1_);

      if (!s2inv_.get())
      {
        s2inv_ = sundials::experimental::N_VectorView(N_VClone(s2));
```